spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -20,26 +20,57 @@ def foobar_action
 end
 
 RSpec.describe Hq::PartnersController, type: :controller do
+
   before :each do
     sign_in instance_double(AdminUser)
-    @partner = build_stubbed :partner
-    expect(Partner).to receive(:find).and_return(@partner)
   end
 
-  specify 'editing renders the edit view' do
-    get :edit, id: @partner.id
-    expect(response).to render_template 'edit'
-  end
+  context '#new and #create' do
+    
+    before :each do
+      @partner = FactoryGirl.build_stubbed(:partner)
+    end
+    
+    specify 'successful create redirects to the show view' do
+      expect_any_instance_of(Partner).to receive(:save) do |_self|
+          _self.id = @partner.id
+          true
+      end
+      post :create, partner: @partner.attributes
+      expect(response).to redirect_to hq_partner_path(@partner.id)
+    end
```

Comment:
I replaced the tests and made sure they passed, but the output of `hq_partner_path(partner_double)` (which happened to be `/hq/partners/%23%5BRSpec::Mocks::InstanceVerifyingDouble:0x772fee6%20@name=%22Partner%20(instance)%22%5D`) made my head hurt a little.